azure/azure_style.yaml

```diff
@@ -52,7 +52,7 @@ jobs:
           versionSpec: "3.8"
       - script: |
           cd ${{ parameters.REPO_NAME }}
-          pip install black==21.6b0
+          pip install -r ../.github/black-requirements.txt
```

Comment:
black currently doesn't checkout `.github`

Reply:
Yeah I guess this requirements file is unnecessary, I can change it back.